.github/rdmo-app-ci-script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+git clone https://$GITHUB_TOKEN@github.com/rdmorganiser/rdmo.git
+# need rdmo only for testing/config/settings
+git clone https://$GITHUB_TOKEN@github.com/rdmorganiser/rdmo-app.git
+
+cd rdmo-app || exit 1
+python3 -m venv env
+source env/bin/activate
+pip install --upgrade pip setuptools
+pip install rdmo"[allauth]"
+
+# install and set-up plugin
+cd - || exit 1
+pip install .
+cp rdmo_radar/sample.local.py rdmo-app/config/settings/local.py
+PLUGIN="$(basename "$(pwd)")"
+PLUGIN_NAME="${PLUGIN/rdmo-plugins-/}"
+
+# set up rdmo-app settings
+# write to rdmo-app/config/settings/__init__.py
+cp rdmo/testing/config/settings/* rdmo-app/config/settings
+cd rdmo-app || exit 1
+mkdir vendor
+# set up instance
+# python manage.py download_vendor_files  # download front-end vendor files
+python manage.py migrate                # initializes the database
+python manage.py setup_groups           # optional: create groups with different permissions
+
+python manage.py loaddata -v 2 "../rdmo/testing/fixtures/users.json"
+python manage.py loaddata -v 2 ../rdmo/testing/fixtures/*
+
+python manage.py check
+
+# function for testing presence of plugin name in a certain django setting
+test_if_settings_contain_plugin () {
```

Comment:
thank you @triole! Ive included your suggestions and refactored the last part into a function. Looks good to you?

Reply:
Hmm, this is just testing that the plugin is in settings which is rather pointless, since the settings are part of the test setup. I think a simple `./manage.py check` would suffice, right?